NuRadioReco/utilities/trace_utilities.py

```
@@ -427,7 +427,8 @@ def split_trace_noise_rms(trace, segments=4, lowest=2):
         The mean rms of the lowest few segment rms values
     """
     split_array = np.array_split(trace, segments)
-    rms_of_splits = np.std(split_array, axis=1)
+    split_array = np.array(split_array, dtype="object") #Objectify dtype to allow timetraces indivisible by amount of segments 
+    rms_of_splits = [np.std(split) for split in split_array]
```

This is an elegant fix. But I was wondering if it is not better to raise an error and in cases where the traces are not dividable by 4 we have to change the number?

---

Having some sort of print/log for the indivisibility does seem like a good idea indeed
I'd be more in favor of raising a warning then as I'd like to leave the possibility of splitting the timetrace in as many segments as one needs. In case of enough samples per event trace, having some segments have one more sample shouldn't make a big difference anyhow I believe